build/ci-pipeline.yml

```diff
@@ -0,0 +1,160 @@
+# DESCRIPTION: 	
+# Builds, tests, and packages the solution for all PR requests.	
+
+name: $(SourceBranchName)-$(Date:yyyyMMdd)$(Rev:-r)
+trigger: none
+
+variables:
+- template: ci-variables.yml
+- template: build-variables.yml
+
+stages:
+- stage: BuildUnitTests
```

Comment:
Just trying to understand where this build will be triggered from? Will it happen from master branch after a PR gets merged in?

Reply:
Yes, the trigger is set up in devops to run when something is merged to master.